x-pack/plugins/endpoint/public/applications/endpoint/store/policy_details/index.test.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { PolicyDetailsState } from '../../types';
+import { applyMiddleware, createStore, Dispatch, Store } from 'redux';
+import { policyDetailsReducer, policyDetailsMiddlewareFactory, PolicyDetailsAction } from './index';
+
+import { coreMock } from '../../../../../../../../src/core/public/mocks';
+import { CoreStart } from 'kibana/public';
+import { DepsStartMock, depsStartMock } from '../../mocks';
+import { selectPolicyConfig, selectWindowsEventing } from './selectors';
+import { clone } from '../../models/policy_details_config';
+
+describe('policy details: ', () => {
+  let fakeCoreStart: jest.Mocked<CoreStart>;
+  let depsStart: DepsStartMock;
+  let store: Store<PolicyDetailsState>;
+  let getState: typeof store['getState'];
+  let dispatch: Dispatch<PolicyDetailsAction>;
+
+  beforeEach(() => {
+    fakeCoreStart = coreMock.createStart({ basePath: '/mock' });
+    depsStart = depsStartMock();
+    store = createStore(
+      policyDetailsReducer,
+      applyMiddleware(policyDetailsMiddlewareFactory(fakeCoreStart, depsStart))
```

Comment:
Is it necessary to use this middleware?

Reply:
it will be used once the backend is hooked up which should be in the next pr

Reply:
Consider adding only code that is used. When you hook up the middleware, consider an API integration test or functional test instead of trying to mock/test the middleware in a unit test.